### PR TITLE
Version Tether: Continuous deployment and versioning

### PR DESCRIPTION
**Pull request by Shadowfiend**
This PR picks up the work in keep-core, keep-ecdsa, tbtc, and tbtc.js to
publish new npm packages on contract migration. It pulls the latest version
of tbtc.js with latest contract references by updating the version string
referencing tbtc.js to properly integrate with tbtc.js's versioning, and updates
the deployment build to resolve the latest version of tbtc.js as well.

### CODE REVIEW

**Comment by Shadowfiend** [comment]
Ok, successful deploy, successful deposits, successful redemption, all ready to merge here once keep-network/tbtc.js#15 lands.

File: .circleci/config.yml [hunk 1]
```diff
@@ -12,12 +12,28 @@ orbs:
...
 jobs:
+  resolve_latest:
+    executor: docker-node
+    steps:
+      - checkout
+      - run:
+          name: Resolve latest tbtc.js
+          command: |
+            npm update @keep-network/tbtc.js
+            echo "Resolved tbtc.js version " $(jq '.dependencies["@keep-network/tbtc.js"]')
```
Comment (liamzebedee) [inline .circleci/config.yml:23]: Where is `jq` getting its JSON from here?

Comment (Shadowfiend) [reply .circleci/config.yml:23]: I actually don't need to do this, because `npm update` shows the version when it does a bump.

Comment (Shadowfiend) [reply .circleci/config.yml]: Fixed.

**Pull request merged**